spec/stat_tracker_spec.rb

```diff
@@ -344,6 +344,19 @@
        
             expect(@stat_tracker.win_loss_ratio("20122014")).to eq(expected)
         end
+        it "can factor in tied games as well" do
```

Comment:
Thank you for completing the final coverage of this tricky method.
Again, good use of mocking.